test-tx-meta-baseline-next/InvokeHostFunctionTests.json

```diff
@@ -1371,6 +1372,5 @@
 		"bKDF6V5IzTo="
 	],
 	"temp entry eviction" : [ "bKDF6V5IzTo=", "bKDF6V5IzTo=" ],
-	"transaction validation diagnostics" : [ "bKDF6V5IzTo=" ],
-	"version test" : [ "766L+TYsWqA=" ]
```

Comment:
Why did this disappear?

Reply:
`version test` is only compiled in if `ENABLE_PROTOCOL_UPGRADE_VIA_SOROBAN_ENV_HOST_PREV` is enabled. I must've mistakenly commited this line in when I added the test case.